packages/charrua-core/charrua-core.0.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+name: "charrua-core"
+version: "0.1"
+maintainer: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+authors: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+homepage: "https://github.com/haesbaert/charrua-core"
+bug-reports: "https://github.com/haesbaert/charrua-core/issues"
+license: "ISC"
+dev-repo: "https://github.com/haesbaert/charrua-core.git"
+available: [ocaml-version >= "4.02"]
+build: [
+  ["sh" "build.sh"]
+]
+depends: [
```

Comment:
I'm about to submit a package that will suffer the same issues, but next version it should be fixed. Thanks for the pointers, I struggled a bit with the build system.

https://github.com/haesbaert/charrua-unix/issues/1